gulpfile.js

```diff
@@ -1,61 +1,92 @@
-var browserify = require('gulp-browserify');
-var clean = require('gulp-clean');
+
+/**
+ * Dependencies.
+ */
+
 var gulp = require('gulp');
-var jsx = require('gulp-jsx');
+var del = require('del');
+var react = require('gulp-react');
+var browserify = require('gulp-browserify');
 var reactify = require('reactify');
-var run = require('gulp-run');
-var runSequence = require('run-sequence');
-var shell = require('gulp-shell');
 var yaml = require('gulp-yaml');
 
+/**
+ * Build Paths.
+ */
+
 var paths = {
-  js: ['client/**/*.js'],
-  jsx: ['client/**/*.jsx'],
-  queries: ['client/**/*.yaml']
+
+  // src
+  appJs: ['src/app/**/*.js'],
+  serverJs: ['src/server/**/*.js'],
+  jsx: ['src/app/**/*.jsx'],
+  queries: ['src/server/**/*-queries.yaml'],
+
+  // dist
+  distPrivateServer: 'dist/private/server',
+  distPrivateApp: 'dist/private/app',
```

Comment:
Seems like the names might be too implementation-driven. How about just `distServer` and `distPrivate`?